cloud/pkg/admissioncontroller/admission.go

```diff
@@ -222,31 +223,49 @@ func (ac *AdmissionController) registerWebhooks(opt *options.AdmissionOptions, c
 		},
 	}
 
-	return registerValidateWebhook(ac.Client.AdmissionregistrationV1beta1().ValidatingWebhookConfigurations(),
-		[]admissionregistrationv1beta1.ValidatingWebhookConfiguration{deviceModelCRDWebhook})
-}
+	objectSelector, err := metav1.ParseToLabelSelector("app-role.kubernetes.io=edge")
```

Comment:
The label in edge app here should reflect the migration info, how about:
```suggestion
	objectSelector, err := metav1.ParseToLabelSelector("app-offline.kubeedge.io=autonomy")
```
cc @kevin-wangzefeng

Reply:
+1 to `app-offline.kubeedge.io=autonomy`
And please define `app-offline.kubeedge.io=autonomy` as a const.